src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -137,4 +141,33 @@ void lostFeeCancellationDoesNotTriggerMarkingItemAsLostAndPaid() {
     assertThat(itemsClient.getById(item.getId()).getJson(), isAvailable());
   }
 
+  @Test
+  void shouldCancelItemFeeActualCostAndProcessingFee() {
+    final double itemFeeActualCost = 15.00;
+    final double itemProcessingFee = 10.00;
+    final UUID servicePointId = servicePointsFixture.cd1().getId();
+    UserResource user = usersFixture.charlotte();
+    feeFineOwnerFixture.ownerForServicePoint(servicePointId);
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .withName("Lost item fee actual cost policy")
+        .chargeProcessingFeeWhenDeclaredLost(itemProcessingFee)
+        .withActualCost(itemFeeActualCost)).getId());
```

Comment:
Can you please try reusing Yusuf's method from:https://github.com/folio-org/mod-circulation/pull/1141/files#diff-6bf88f66480f263533b5d35a97c1f8b348d6f563d592bd535d4d3073e2358209